src/renderer/views/TechnologyPicker/TechnologiesPicker.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import StyledSwitch from '../../components/Switch/Switch';
+import './TechnologiesPicker.scss';
+
+const TechnologiesPicker = () => {
+  const { t, i18n } = useTranslation();
+  let [useSCSS, changeUseSCSS] = useState(true);
+  let [useTP, changeTP] = useState(true);
+
+  return (
+    <div className='container'>
+      <h2>{t('technology-picker.message')}</h2>
+
+      <div className='switches'>
+        <div className='switch'>
+          <StyledSwitch
+            size='small'
+            checked={useSCSS}
+            onChange={() => {
+              changeUseSCSS((useSCSS = !useSCSS));
+            }}
+          />
+          <p className='inline'> SCSS</p>
+        </div>
+        <div>
+          <StyledSwitch
+            size='small'
+            checked={useTP}
+            onChange={() => {
+              changeTP((useTP = !useTP));
+            }}
+          />
```

Comment:
In the spirit of using redux the `onChange` handlers should be changed here as well. However, the use of the `changeUseSCCS` and `changeTP` was also wrong in itself. The function accepts a param. What you're doing here is assigning to `useSCSS` witht the `=` operator and then the result of that assignment is passed to the function. It's a weird thing in js that this works. The proper way to use the `useState`'s set function to change the value based on the previous one would be the follwing:

```ts
// The set function either accepts a value or a function that accepts the previous state value and returns a new one.
changeUseSCSS((prevUseSCCS) => !prevUseSCSS)
```

We're using redux for managing the state though so the `onClick` handlers should be changed to utilize that as well.
Therefore, you could define two handlers in the component as follows

```ts
// The properties selected from redux in the comment above
const { useSass, useTypescript } = useSelector(newPublication);
// We need to create a redux dispatch first
const dispatch = useDispatch();

const toggleSass = () => {
  // We can then dispatch the setPublicationField action from addPublicationSlice to update the useSass
  dispatch(setPublicationField({ field: 'useSass', value: !useSass }));
};

const toggleTypescript = () => {
  // Same thing can be done for the useTypescript property
  dispatch(setPublicationField({ field: 'useTypescript', value: !useTypescript }));
};
```

These can then be passed to the switches

```tsx
<StyledSwitch
  size='small'
  checked={useSass}
  onChange={toggleSass}
/>
```